plugins/lang/adminslots.txt

```diff
@@ -1,5 +1,7 @@
 [en]
 DROPPED_RES = Dropped due to slot reservation
+CVAR_RESERVATION = Amount of slots to reserve
+CVAR_HIDESLOTS = If you set this to 1, you can hide slots on your server.^nIf server "full" of public slots and slots hidden, you must manually connect with connect console command.
```

Comment:
> If server "full" of public slots and slots hidden, you must manually connect with connect console command.

=>

> If the server's public and hidden slots are full, you must manually connect with the "connect" console command.

Reply:
I used the original text, but yes, I can change it.